scripts/speaker_tasks/diarization_finetuning_manifest.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import random
+
+from nemo.collections.asr.parts.utils.manifest_utils import create_segment_manifest
+
+random.seed(42)
+
+
+"""
+This scipt converts a scp file where each line contains
+<absolute path of wav file>
+to a manifest json file.
+Args:
+--scp: scp file name
+--id: index of speaker label in filename present in scp file that is separated by '/'
+--out: output manifest file name
+--split: True / False if you would want to split the  manifest file for training purposes
+        you may not need this for test set. output file names is <out>_<train/dev>.json
+        Defaults to False
+--create_chunks: bool if you would want to chunk each manifest line to chunks of 3 sec or less
+        you may not need this for test set, Defaults to False
+"""
+
+
+def main(input_manifest_path, output_manifest_path, window, shift, step_count, deci):
+    create_segment_manifest(input_manifest_path, output_manifest_path, window, shift, step_count, deci)
+
+
+if __name__ == "__main__":
```

Comment:
Can you change the name of this file to `create_msdd_train_dataset.py`?
Using git mv  `diarization_finetuning_manifest.py` `create_msdd_train_dataset.py`.
It's my bad that I didn't change this temporary name.